murfi-rt-PyProject/scripts/launch_murfi.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/sh
```

Comment:
maybe add a safety option that checks if a folder/session/run already exists to avoid overwriting in case of typos?

Reply:
agreed -- I think that's a good next step